chan-plugin/chan-plugin.php

```diff
@@ -0,0 +1,50 @@
+<?php /**
+ * Plugin Name: chan-plugin
```

Comment:
Plugin name appears in frontend should be in human readable format.
Reference for Plugin Header
https://developer.wordpress.org/plugins/plugin-basics/header-requirements/

Reply:
Plugin name has been changed